metadata-ingestion/src/datahub/emitter/mcp_builder.py

```diff
@@ -81,6 +81,9 @@ def guid_dict(self) -> Dict[str, str]:
     def property_dict(self) -> Dict[str, str]:
         return self.dict(by_alias=True, exclude_none=True)
 
+    def as_urn(self) -> str:
+        return make_container_urn(guid=self.guid())
```

Comment:
So are all these `DatahubKey`s really container keys? Can we call them that? `PlatformKey` makes me think we're creating a key for a platform... but really we're creating a key for a container that contains platform information.

Reply:
agree that it's confusing - going to rename PlatformKey to ContainerKey + add some docs